src/translations/en/languages.json

```diff
@@ -14,5 +14,6 @@
   "spa": "Spanish",
   "swe": "Swedish",
   "ger": "German",
-  "mis": "Other"
+  "mis": "Other",
+  "und": "Unknown"
```

Comment:
`und` er et språk, derfor brukte jeg bare den her for oss også, selv om det ikke er mulig å sette den verdien på registreringen

Reply:
er und egentlig `undefined` eller? virker rart at det står for `unknown`?

Reply:
und står for er undefined i ISO-standarden ja, samme som mis egentlig står for miscellaneous. Syns Unknown/Ukjent var et mer beskrivende navn for brukeren